src/mixins/keyboard-shortcut-mixin.ts

```diff
@@ -0,0 +1,26 @@
+import { LitElement } from "lit-element";
+import { Constructor } from "../types";
+
+export const KeyboardShortcutMixin = <T extends Constructor<LitElement>>(
+  superClass: T
+) =>
+  class extends superClass {
+    private _keydownEvent = (event: KeyboardEvent) => {
+      if ((event.ctrlKey || event.metaKey) && event.key === "s") {
+        event.preventDefault();
+        this.saveCallback();
```

Comment:
I think the name should be a bit clearer, maybe
```suggestion
        this.handleKeyboardSave();
```

Reply:
Done! Good suggestion.